packages/koa-shopify-graphql-proxy/src/shopify-graphql-proxy.ts

```diff
@@ -5,10 +5,10 @@ export const PROXY_BASE_PATH = '/graphql';
 export const GRAPHQL_PATH_PREFIX = '/admin/api';
 
 export type ApiVersion =
-  | '2020-10'
-  | '2021-01'
-  | '2021-04'
-  | '2021-07'
```

Comment:
Removing existing values is a breaking API change. Please add these values, but don't remove any existing ones. If you do this then you can revert your changes to the test files too.